### PR TITLE
Add support for (un)linked inventory statuses

### PR DESCRIPTION
**Pull request by honza**
This is a simple inventory item component that takes a count of OK
elements, and a count of Not-OK elements, and displays the appropriate
status icon.  This is useful for inventory items which aren't
trivial (i.e. handled by the InventoryItem component), but don't have a
resource backing them.

### CODE REVIEW

File: frontend/public/components/dashboard/inventory-card/inventory-item.tsx [hunk 1]
```diff
@@ -123,6 +135,11 @@ type InventoryItemProps = {
 type StatusProps = {
   groupID: InventoryStatusGroup | string;
   count: number;
+}
+
+type LinkStatusProps = {
```
Comment (rawagner) [inline frontend/public/components/dashboard/inventory-card/inventory-item.tsx]: could be
```suggestion
type LinkStatusProps = StatusProps & {
  statusIDs: string[];
  kind: K8sKind;
  namespace?: string;
  filterType?: string;
}
```

Comment (honza) [reply frontend/public/components/dashboard/inventory-card/inventory-item.tsx]: Done

File: frontend/public/components/dashboard/inventory-card/inventory-item.tsx [hunk 2]
```diff
@@ -69,7 +69,7 @@ export const InventoryItem: React.FC<InventoryItemProps> = ({ isLoading, singula
   );
 };
 
-const Status: React.FC<StatusProps> = React.memo(({ groupID, count, statusIDs, kind, namespace, filterType}) => {
+const LinkStatus: React.FC<LinkStatusProps> = React.memo(({ groupID, count, statusIDs, kind, namespace, filterType}) => {
```
Comment (spadgett) [inline frontend/public/components/dashboard/inventory-card/inventory-item.tsx]: nit: I would call this `StatusLink`. The other way sounds like it's the status of the link.

Comment (honza) [reply frontend/public/components/dashboard/inventory-card/inventory-item.tsx]: Agreed, done

File: frontend/public/components/dashboard/inventory-card/inventory-item.tsx [hunk 3]
```diff
@@ -60,7 +60,18 @@ export const InventoryItem: React.FC<InventoryItemProps> = React.memo(
...
+  const groupIcon = statusGroupIcons[groupID] || statusGroupIcons[InventoryStatusGroup.NOT_MAPPED];
+  return (
+    <div className="co-inventory-card__status">
+      {groupIcon}
+      <span className="co-inventory-card__status-text">{count}</span>
+    </div>
+  );
+});
+
+const StatusLink: React.FC<StatusLinkProps> = React.memo(({ groupID, count, statusIDs, kind, namespace, filterType}) => {
```
Comment (vojtechszocs) [inline frontend/public/components/dashboard/inventory-card/inventory-item.tsx]: Nit: space after `filterType` (or remove space before `groupID`)

**Comment by vojtechszocs** [comment]
LGTM just one nit-pick.

**Pull request merged**